manifests/params.pp

```diff
@@ -309,8 +309,8 @@
     $ssl_certs_dir    = '/usr/local/etc/apache24'
     $passenger_conf_file = 'passenger.conf'
     $passenger_conf_package_file = undef
-    $passenger_root   = '/usr/local/lib/ruby/gems/1.9/gems/passenger-4.0.10'
-    $passenger_ruby   = '/usr/bin/ruby'
+    $passenger_root   = '/usr/local/lib/ruby/gems/2.0/gems/passenger-4.0.58'
```

Comment:
All builds on all supported FreeBSD versions since Ruby 2.0 was made the default (2014-10-08). 

The passenger version is also a fast moving target, but I have to think more on how to solve this (maybe also not here. but in the package).